### PR TITLE
Update plant & water their seeds link

### LINKED ISSUE #331: Old Liquidity Pool Link on Wiki
This link reading 'plant & water their seeds' on the wiki page https://wiki.metagame.wtf/docs/how-does-it-work/xp

Leads to https://beta.pools.balancer.exchange/#/pool/0xea05a15dbce2eb543ffda16950e95b2bd2e40d0e/swaps

And needs to be updated to 

https://polygon.balancer.fi/#/pool/0x8a8fcd351ed553fc75aecbc566a32f94471f302e000100000000000000000081/invest

image

Also the link reading 'available for sale'

Goes to our mainnet Seed at app.balancer.fi

And maybe should lead to our Polygon seed or also to our Polygon seed at

https://polygon.balancer.fi

We may want to include the token address to import Seed as well?

image

### PR DESCRIPTION
**Pull request by vidvidvid**
Closes #331

### CODE REVIEW

**Comment by Misanth420** [comment]
[update plant & water their seeds link](https://github.com/MetaFam/metagame-wiki/pull/340/commits/a01a0d9f6a355bdbf0526e58f4d7af1e7d40d634) In regards to the line that claims it requires a supply of eth and seed- Do we remove it completely, rephrase it to include RAI or something else?

**Comment by peth-yursick** [comment]
Ahh the perils of duplicating info... I'd say just remove altogether & link to the Seeds page which explains how to become a patron - the problem is that the page is currently in notion so it will be outdated again in a week or two 😅

**Comment by Baconaetor** [comment]
@vidvidvid Can you change the wording to include RAI? So requires Seed, Ethereum, and RAI? Then I'll merge this in.

**Comment by vidvidvid** [comment]
but i did this one change @Baconaetor hope everything is fine now.

**Review by Baconaetor — APPROVED** [review]
Looks good.
